tests/MongoDB.Driver.Legacy.Tests/MongoDatabaseTests.cs

```diff
@@ -322,6 +323,7 @@ public void TestGetCollection()
             Assert.Equal(_database.Name + "." + collectionName, collection.FullName);
             Assert.Equal(collectionName, collection.Name);
             Assert.Equal(_database.Settings.WriteConcern, collection.Settings.WriteConcern);
+            Assert.Equal(_database._operationExecutor(), collection.OperationExecutor);
```

Comment:
Test that the MongoCollection inherits the _operationExecutor from the MongoDatabase.